app/resources/start.html.template

```diff
@@ -55,15 +52,15 @@
       js.src = "//connect.facebook.net/ja_JP/sdk.js#xfbml=1&version=v2.8";
       fjs.parentNode.insertBefore(js, fjs);
       }(document, 'script', 'facebook-jssdk'));</script>
-      <div class="fb-share-button" data-href="https://google.org/personfinder/global/home.html" data-layout="button_count" data-size="small" data-mobile-iframe="true" line-height="0"><a class="fb-xfbml-parse-ignore" target="_blank" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fgoogle.org%2Fpersonfinder%2Fglobal%2Fhome.html&amp;src=sdkpreparse">Share</a></div>
+      <div class="fb-share-button" data-href="{{env.url}}" data-layout="button_count" data-size="small" data-mobile-iframe="true" line-height="0"><a class="fb-xfbml-parse-ignore" target="_blank" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fgoogle.org%2Fpersonfinder%2Fglobal%2Fhome.html&amp;src=sdkpreparse">Share</a></div>
 
+      {# twitter share button #}
```

Comment:
Very nit-pick, but capitalize "twitter" (to "Twitter") for consistency with "Facebook". Same for below. Would be better to use the official name "Google+" as well.